heron/spi/tests/java/com/twitter/heron/spi/packing/PackingPlanTest.java

```diff
@@ -25,7 +25,7 @@
 import com.twitter.heron.spi.common.Constants;
 
 public class PackingPlanTest {
-  public static PackingPlan generatePacking(Map<String, List<String>> basePacking) {
+  private static PackingPlan generatePacking(Map<String, List<String>> basePacking) {
     Resource resource =
```

Comment:
Yes, it's still used in the ContainerPlan and the PackingPlan objects below. I just changed to a resource for each instance that was unique, so I could assert on that.